smartcontractkit/external-adapters-js
# Chore/LOG_LEVEL warning

## Closes SC-56144

## Description

Log a warning on startup when LOG_LEVEL if set to DEBUG or TRACE to convey the risks involved with using higher levels

## Changes

- Added warning log for LOG_LEVEL risks on adapter startup

## Steps to Test

1. Set LOG_LEVEL to DEBUG or TRACE
2. Start an adapter
3. Confirm a warning is logged on startup mentioning the risk of using higher log levels

## Quality Assurance
